clustering.Rmd

```diff
@@ -45,95 +49,147 @@ use the subgroups to generate new questions about the data and follow up with a
 predictive modelling exercise. In this course, clustering will be used only for
 exploratory analysis, i.e., uncovering patterns in the data.  
 
-Note that clustering is a fundamentally different kind of task than classification or
-regression. In particular, both classification and regression are *supervised tasks* 
-\index{classification}\index{regression}\index{supervised} where 
-there is a *predictive target* (a class label or value), and we
-have examples of past data with labels/values that help us predict those of
-future data. By contrast, clustering is an *unsupervised task*, \index{unsupervised} as we are
-trying to understand and examine the structure of data without any labels to
-help us. This approach has both advantages and disadvantages. Clustering
-requires no additional annotation or input on the data. For example, it would
-be nearly impossible to annotate all the articles on Wikipedia with human-made
-topic labels. However, we can still cluster the articles without this
-information to find groupings corresponding to topics automatically. 
-
-However, because there is no predictive target, it is not as easy to evaluate
+Note that clustering is a fundamentally different kind of task 
+than classification or regression. 
+In particular, both classification and regression are *supervised tasks* 
+\index{classification}\index{regression}\index{supervised} 
+where there is a *response variables* (a category label or value), 
```

Comment:
grammar "response variables" plural should be singular

Reply:
done